src/Files.App/Data/Models/AppModel.cs

```diff
@@ -119,10 +119,13 @@ public string PCloudDrivePath
 
 		/// <summary>
 		/// Gets or sets a value indicating the AppWindow DPI.
+		/// TODO update value if the DPI changes
 		/// </summary>
-		public float AppWindowDpi
+		private float appWindowDPI = InteropHelpers.GetDpiForWindow(MainWindow.Instance.WindowHandle) / 96f;
```

Comment:
According to the coding guideline, this should be _AppWindowDPI - a field backing a property.

Reply:
I updated the other properties in AppModel as well in order to be consistent

Reply:
Thank you.